### ISSUE TITLE
Remove periods from ends of error comments...

### ISSUE THREAD

**Issue by citibeth** [LLNL/spack#455]
...when the errors end in a file name or path.  The periods add nothing (practical), but they do make it harder to cut-n-paste paths.  For example:

```
==> Already staged curl-7.46.0-64oocwwozzc7nfxpgjcqsam22islbdkf in /home/me/spack/var/spack/stage/curl-7.46.0-64oocwwozzc7nfxpgjcqsam22islbdkf.
```

**Comment by eschnett** [comment]
+1

**Comment by tgamblin** [comment]
I like this idea.  Feel free to send along a PR...

**Comment by citibeth** [comment]
Assign it to me, if you like.

**Comment by tgamblin** [comment]
Apparently I have to add you to a team to assign issues.  Invitation sent.

**Comment by citibeth** [comment]
Done: https://github.com/LLNL/spack/pull/489

**Comment by tgamblin** [comment]
Thanks!  @citibeth: FYI: if you put, e.g., "Fixes #455" in the PR subject, closing this happens automatically on merge.
